ceph/ceph_admin/apply.py

```diff
@@ -30,3 +41,51 @@ def apply(self, role, command, placements):
                 daemon=role,
                 ids=placements,
             )
+
+    def orch_remove(self, **config):
```

Comment:
Unlike `apply`, do you see the need to pass a `dict`?

Can we have `service_name` & `check_status=True` arguments/parameters only?